samples/Client/DaprClient/Program.cs

```diff
@@ -48,28 +48,41 @@ public static async Task Main(string[] args)
             // Delete State
             await DeleteStateAsync(client);
 
-            #region Service Invoke - Required RoutingService
-            //// This provides an example of how to invoke a method on another REST service that is listening on http.
-            //// To use it run RoutingService in this solution.
-            //// Invoke deposit operation on RoutingSample service by publishing event.      
+            // Invoke deposit operation on ControllerSample or RoutingSample or GrpcServiceSample service by publishing event.      
+
+            await PublishDepositeEventAsync(client);
 
-            //await PublishDepositeEventToRoutingSampleAsync(client);
+            await Task.Delay(TimeSpan.FromSeconds(1));
 
-            //await Task.Delay(TimeSpan.FromSeconds(1));
+            #region Service Invoke - Required RoutingService
+            // This provides an example of how to invoke a method on another REST service that is listening on http.
+            // To use it run RoutingService in this solution.
 
-            //await DepositUsingServiceInvocation(client);
+            //Invoke deposit operation on RoutingSample service by POST.
+            //await InvokeDepositServiceOperationAsync(client);
 
-            ////Invoke deposit operation on RoutingSample service by POST.
+            //Invoke withdraw operation on RoutingSample service by POST.
             //await InvokeWithdrawServiceOperationAsync(client);
 
-            ////Invoke deposit operation on RoutingSample service by GET.
+            //Invoke balance operation on RoutingSample service by GET.
             //await InvokeBalanceServiceOperationAsync(client);
             #endregion
 
+            #region Service Invoke via GRPC - Required GrpcServiceSample
```

Comment:
Can the client be added under samples/AspNetCore/GrpcServiceSample? This is adding a dependency to start the service before running this client sample.

Reply:
Actually, original code also depends on running of RoutingService. 
I think best solution is add comment for method calling.

Reply:
I think the better solution would be to add a separate project for the GRPC client OR make this program take an argument that switches between what client code it's running AND refactor this big main method into a few separate files.

Right now this is pretty hard to navigate and understand. It's worthwhile to optimize samples to be easy for users to find what they want.

Reply:
I added two parameters to switch using of routing service or grpcsample service.